test/import.bats

```diff
@@ -390,6 +390,79 @@ eigth:
     [ -f /dir/file2 ]
     [ -f /dir/files/file3 ]
 EOF
+    stacker build
+}
+
+@test "import with dir contents" {
+  mkdir folder1
+  touch folder1/file1
+  mkdir folder1/subfolder2
+  touch folder1/subfolder2/subfile1
+  cat > stacker.yaml <<EOF
+first:
+  from:
+    type: oci
+    url: $CENTOS_OCI
+  import:
+  - path: folder1/
+    dest: /folder1/
+  run: |
+    [ -f /folder1/file1 ]
+
```

Comment:
If I'm reading things correctly, then these two snippets do the same thing:

 *
   ```yaml
   import:
    - path: folder1
      dest: /
   ```

and

 *
   ```yaml
   import:
      - path: folder1
        dest: /folder1/
   ```

I find it odd that the trailing '/' on a 'path' element only means something if the 'dest' is /.

Reply:
It means that folder1 is a dir and everything in src goes under folder1/ as a child unambiguously.

Reply:
I still think that rsync semantics make the most sense.

rsync -va src/ dest/ will copy contents of src/ into dest/.
rsync -va src dest/ will copy src as a directory under dest/ (plus of course all its contents).